openshift/backup.dc.yml

```diff
@@ -410,19 +410,19 @@ parameters:
     displayName: Resources CPU Request
     description: The resources CPU request (in cores) for this build.
     required: true
-    value: "50m"
+    value: "25m"
   - name: CPU_LIMIT
     displayName: Resources CPU Limit
     description: The resources CPU limit (in cores) for this build.
     required: true
-    value: "150m"
+    value: "50m"
   - name: MEMORY_REQUEST
     displayName: Resources Memory Request
     description: The resources Memory request (in Mi, Gi, etc) for this build.
     required: true
-    value: 256Mi
+    value: 48Mi
   - name: MEMORY_LIMIT
     displayName: Resources Memory Limit
     description: The resources Memory limit (in Mi, Gi, etc) for this build.
     required: true
-    value: 512Mi
\ No newline at end of file
+    value: 32Mi
```

Comment:
I think the limit has to be less than the request?

Reply:
Good catch haha